packages/docs-page/components.ts

```diff
@@ -1,9 +1,10 @@
 import defaultMdxComponents from '@hashicorp/platform-docs-mdx'
+import { MDXProviderComponentsProp } from '@mdx-js/react'
 
 export default function generateComponents(
-  productName,
-  additionalComponents = {}
-) {
+  productName: string,
+  additionalComponents: MDXProviderComponentsProp = {}
+): Record<string, string> {
```

Comment:
`defaultMdxComponents` returns a map of `string` to components I think 💭 

https://github.com/hashicorp/nextjs-scripts/blob/main/packages/docs-mdx/index.jsx#L11

Reply:
I'll do:

```ts
Record<string, (p: any) => JSX.Element>
```